core/renderers/measurables/row_elements.js

```diff
@@ -95,7 +95,7 @@ Blockly.blockRendering.Field = function(constants, field, parentInput) {
   Blockly.blockRendering.Field.superClass_.constructor.call(this, constants);
   this.field = field;
   this.isEditable = field.isCurrentlyEditable();
-  this.flipRtl = field instanceof Blockly.FieldImage && field.getFlipRtl();
+  this.flipRtl = field.getFlipRtl && field.getFlipRtl();
```

Comment:
.getFlipRtl() should be moved to Field, and Field.prototype.flipRtl_ should just be set to false.

Reply:
If it gets moved it might need a doc issue as well.